modules/client/examples/index.ts

```diff
@@ -3,9 +3,9 @@
 title: Getting Started
 ---
 
-## Getting Started with the Aragon OSx SDK
+## Getting Started With the Aragon OSx SDK
```

Comment:
The official brand is `Aragon SDK`

```suggestion
## Getting Started with the Aragon SDK
```

Reply:
Fixed in 6fc88cb2d013aac7f2c766765eb305afee585ffcand used https://capitalizemytitle.com/ for capitalization